src/inmanta/data/dataview.py

```diff
@@ -559,6 +583,7 @@ def construct_dtos(self, records: Sequence[Record]) -> Sequence[model.LatestRele
                 requires=json.loads(resource["attributes"]).get("requires", []),
             )
             for resource in records
+            if resource["resource_id"]  # filter out bad joins
```

Comment:
So we accept that the count may be inaccurate?

Reply:
No, I see you added the trimming of the table. Why do we need this then? Is it purely "in case of some unexpected state"?

Reply:
no, if there is no released version